docs/alerts/monitors/create-monitor.md

```diff
@@ -176,17 +176,22 @@ Sumo Logic automatically resolves the incident when the resolution condition is
 | Threshold type | How you want the value compared. Select either greater than, greater than or equal, less than or equal, or less than. |
 | Threshold | The value against which the resolution will be evaluated. You can specify any valid numeric value. |
 
-**Alert and recovery window**
+#### Alert and recovery window
 
-This setting controls the behavior of alert/recovery for sparse/intermittent/incomplete data sources.
+This setting affects both the alert generation logic and the alert recovery logic.
 
 ![metrics alert datapoints.png](/img/monitors/minimum-datapoints.png)
 
-`Alert and recovery require a minimum of <Count> data points for “at all times” evaluation windows`
+`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
 
 | Parameter | Description |
 |:--|:--|
-| Count | The minimum number of data points needed in alert and recovery `Time Range` for alert to trigger and resolve respectively. |  
+| Count | The minimum number of data points required within the 5-minute window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).
```

Comment:
The window might not always be 5 mins. `The minimum number of data points required within the configured window.